modules/sso.php

```diff
@@ -732,6 +724,29 @@ function handle_login() {
 		add_action( 'login_message', array( $this, 'cant_find_user' ) );
 	}
 
+	static function sso_redirect( $user = null ) {
```

Comment:
Does this method have to be static? I can see it being called as an object method.